ccm_web/client/src/utils/handleErrors.ts

```diff
@@ -23,7 +23,7 @@ class UnauthorizedError extends Error {
   constructor () {
     super(
       'You are not authorized to perform that action. ' +
-      'You may need to authenticate differently.'
+      'If you believe this is an error, try re-launching the application or contact support.'
```

Comment:
Note that I removed the "If ..." part in my PR. Didn't seen necessary, seems better to be concise here.